playbooks/generic-clean_userfiles.yml

```diff
@@ -0,0 +1,34 @@
+---
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- hosts: localhost
+  gather_facts: no
+  connection: local
+  vars_prompt:
+    - name: "delete_anyway"
+      prompt: "Delete all files in workspace?"
+      default: "no"
+  tasks:
+    - name: Load workspace var
+      include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+      when: delete_anyway | bool
+    - name: Deleting files in workspace
+      file:
+        path: "{{ socok8s_workspace }}"
+        state: absent
+      when:
+        - "delete_anyway | bool"
```

Comment:
(nit) could be done as a block rather than having "delete_anyway" on both tasks.